modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusModule.java

```diff
@@ -180,10 +180,13 @@ private void doStartNode(final String defaultConfig) {
             JSONObject customConfig = new JSONObject(defaultConfig);
             JSONObject jsonConfig = new JSONObject(config);
             String gethLogFileName = "geth.log";
+            String dataDir = root + customConfig.get("DataDir");
```

Comment:
Is this always present?

Reply:
That's not something introduced in this PR, i only moved it to variable